frontend/scss/modules/_badge.scss

```diff
@@ -8,7 +8,7 @@
 	background: #d4d4d4;
 	width: 26px;
 	height: 26px;
-	margin: 0 $space-2;
+	margin: 0 $space-2 20px 0;  // Tmp fix badge
```

Comment:
should I merge it or do you first want to do a proper fix?

Reply:
We can merge it, I added it to mark temporary solution while waiting for new design.